MoltenVK/MoltenVK/API/mvk_datatypes.h

```diff
@@ -34,6 +34,8 @@ extern "C" {
  *
  * The functions here are used internally by MoltenVK, and are exposed here
  * as a convenience for use elsewhere within applications using MoltenVK.
+ * These functions can only be used if the application is linked directly 
+ * to MoltenVK, and not dynamically through the Vulkan Loader.
```

Comment:
Is this true anymore? Calling any of these functions should result in a call to `getPlatformPixelFormats()`, which should initialize the `_platformPixelFormats` object.

Reply:
It wasn't that. It's that the MoltenVK SDK build hides many of the linkable functions within MoltenVK. That's the purpose of `MVK_PUBLIC_VULKAN_SYMBOL` and `MVK_HIDE_VULKAN_SYMBOLS`. 

Also, there is a strong push towards SDK MoltenVK not being accessed except through supported Vulkan API functions. This is why MoltenVK will be supporting `VK_EXT_layout_settings`. The idea for this is to avoid user confusion and complaints when they can't access MoltenVK functions through the Vulkan layers.

However, the `mvk_datatypes.mm` functions are marked with `MVK_PUBLIC_SYMBOL`, so they will always be exposed, and they don't interact with Vulkan API calls at all, or involve passing layer objects. So in a technical sense, you are accurate, and I've removed the comment.